scripts/js/commands/checkInternalLinks.ts

```diff
@@ -294,17 +290,15 @@ async function determineQiskitLegacyReleaseNotes(): Promise<FileBatch> {
     .map((releaseNotesPath) =>
       releaseNotesPath.split("/").pop()!.split(".").slice(0, -1).join("."),
     )
-    .filter(
-      (version) => +version < 1 && version != "0.45" && version != "0.46",
-    );
+    .filter((version) => +version < 1 && version != "0.46");
```

Comment:
This is inside determineQiskitLegacyReleaseNotes(). 0.46 is now checked as part of `--historical-apis`. I'm not sure why 0.45 wasn't being checked in this batch previously.